setup/main.py

```diff
@@ -628,9 +628,14 @@ def __init_available_gtk_themes(self):
         gtk_theme_path = []
         for path in path_list:
             gtk_theme_path.extend(glob.glob(path + "/*/*/gtk.css"))
+            gtk_theme_path.extend(glob.glob(path + "/*/*/gtk-dark.css"))
```

Comment:
I could not find the differences between Adwaita-dark and Adwaita:dark themes. Probably I think one of "-dark" and ":dark" themes can be deleted on ibus-setup.

Reply:
Well, the majority themes provide the dark variant feature, not providing `-dark`, while some other old themes providing `-dark`, not the dark variant. So I guess it's better to reflect the real situations so that no themes can be missed.

Reply:
What do you think about "Greybird-dark", "Greybird-dark:dark", "Greybird:dark"?

```
% rpm -qf /usr/share/themes/Greybird-dark/gtk-3.0/gtk-dark.css 
greybird-dark-theme-3.22.14-4.fc35.noarch
```

Reply:
I'm OK if you insist on removing the dark variant support. https://github.com/ibus/ibus/commit/79cd18c98b4110cd9511798a71ced5585df1cfdb

Then I'll make a commit to revert that.

Reply:
Done reverting. So mark this review as resolved...